build.zig.zon

```diff
@@ -1,6 +1,10 @@
 .{
     .name = "zig-base32",
     .version = "0.1.0",
```

Comment:
lmk if we need to bump this

Reply:
Sure, let's bump it to `v0.2.0` maybe to signal a breaking change.

Reply:
Do you mind also bumping the zig version [in the CI](https://github.com/jsign/zig-base32/blob/main/.github/workflows/ci.yml) so we can have a green check? 

If you get in a rabbit hole don't worry and I can merge the PR and fix after.